docs(userguide): Reword some part of invitation for the user the check the link (#10677)
Scille/parsec-cloud: docs/userguide/manage_devices.rst

    :align: center
    :alt: Device invitation link

> 3. Send the link to your new device, either by letting the server send an email for you or by other means of communication.
>    Always verify that the link correspond to the Parsec server you want to add a new device to.

Reviewer: ```suggestion
3. Send the link to your new device, either by email or by any other means.
   Always verify that the link correspond to the same Parsec server of your device.
```

.. important::

  If you do not trust the link, do not proceed.

4. Once the verification done, on the new device, you can simply click on the link to start Parsec.
   Otherwise, you can copy the link, start Parsec and select ``Join`` in the welcome page
   (the application should have pre-filled the link with the clipboard content, but you can manually paste it).
   Then you follow the instructions to perform the code exchange between your devices.

.. caution::

>    You should keep an eyes on both devices during the process to not let anyone meddle with it.
>    That include when you enter the link on the new device.

Reviewer: ```suggestion
   Always keep an eye on both devices during the process and not let anyone interfere.
   This includes notably the step of copying the link to the new device.
```
